impl/maven-cli/src/test/java/org/apache/maven/cling/transfer/FileSizeFormatTest.java

```diff
@@ -18,15 +18,33 @@
  */
 package org.apache.maven.cling.transfer;
 
+import java.util.Locale;
+
 import org.apache.maven.api.services.MessageBuilder;
 import org.apache.maven.cling.transfer.FileSizeFormat.ScaleUnit;
 import org.apache.maven.internal.impl.DefaultMessageBuilder;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 class FileSizeFormatTest {
+    // Locale like DE uses "," and not "."
+    // For simplicity’s sake let test in US
+    static Locale original;
```

Comment:
I suggest to use https://junit-pioneer.org/docs/default-locale-timezone/ for this (and yes I'm a maintainer of this project, together with Nicolai Parlog and others)

Reply:
Created https://issues.apache.org/jira/browse/MNG-8459 to follow up, right now am interested in fixing this single failure only.